build-pipeline.yml

```diff
@@ -13,13 +13,23 @@ parameters:
   flake8_exclude: ""
   # space separated list of Python packages that need to be pip installed.
   extra_pip_dependencies: ""
+  # If true, the unit tests will be skipped
+  skip_tests: false
```

Comment:
Our tk-multi-testing app does not have tests, so save the trouble of spinning up many VMs to run tests that do not exist.